packages/flutter/test/material/dropdown_form_field_test.dart

```diff
@@ -195,6 +195,41 @@ void main() {
     expect(value, equals('three'));
   });
 
+  testWidgets('DropdownButtonFormFieldState value changes when option selected',
+      (WidgetTester tester) async {
+    String value = 'one';
+    final GlobalKey<FormFieldState<String>> stateKey =
+        GlobalKey<FormFieldState<String>>();
+
+    await tester.pumpWidget(
+      Builder(
```

Comment:
Is there a reason to wrap `MaterialApp` in a `Builder` here? Otherwise, please remove it